### PR TITLE
feat: quote behavior changes for seller

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/b2b/seller/SellerQuoteActions/SellerQuoteActions.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+
+  expect(submitForApprovalButton).toBeDisabled()
+}
+
+afterEach(() => {
+  cleanup()
+})
+
+describe('[components] - SellerQuoteActions', () => {
+  // it('should disable clear changes button if quote is InReview/Completed status and quote is not in draft mode', () => {
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/SellerQuoteActions/SellerQuoteActions.spec.tsx]: why commented tests?

File: components/b2b/B2BProductDetailsTable/B2BProductDetailsTable.tsx [hunk 2]
```diff
@@ -146,8 +145,7 @@ export default function B2BProductDetailsTable(props: B2BProductDetailsTableProp
                 )}
               </TableCell>
               <TableCell>
-                {QuoteStatus[status as string] === QuoteStatus.InReview ||
-                QuoteStatus[status as string] === QuoteStatus.Completed ||
+                {QuoteStatus[status as string] === QuoteStatus.Completed ||
```
Comment (Kibo-TriptiV) [inline components/b2b/B2BProductDetailsTable/B2BProductDetailsTable.tsx:148]: Can we have these condition on getters and behave based on seller or buyer?

File: components/b2b/seller/SellerQuoteActions/SellerQuoteActions.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+          {QuoteStatus[status as string] === QuoteStatus.InReview && (
+            <>
+              <NoSsr>
+                {!hasDraft && (
+                  <Box>
+                    <LoadingButton
+                      variant="contained"
+                      color="primary"
+                      fullWidth
+                      // disabled={hasDraft as boolean}
```
Comment (Kibo-TriptiV) [inline components/b2b/seller/SellerQuoteActions/SellerQuoteActions.tsx]: remove commented code

**Review by Kibo-TriptiV — CHANGES_REQUESTED** [review]
Please check review comments.

File: components/b2b/B2BProductDetailsTable/B2BProductDetailsTable.tsx (hunk 2 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/B2BProductDetailsTable/B2BProductDetailsTable.tsx:148]: Will do this in main branch

File: components/b2b/seller/SellerQuoteActions/SellerQuoteActions.spec.tsx (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/SellerQuoteActions/SellerQuoteActions.spec.tsx]: removed

File: components/b2b/seller/SellerQuoteActions/SellerQuoteActions.tsx (hunk 3 above)
Comment (kibo-chandradeeptalaha) [reply components/b2b/seller/SellerQuoteActions/SellerQuoteActions.tsx]: done

**Review by Kibo-TriptiV — APPROVED** [review]

**Pull request merged**